ilastik/shell/gui/startShellGui.py

```diff
@@ -73,7 +73,7 @@ def _applyStyleSheet(app):
     Apply application-wide style-sheet rules.
     """
     styleSheetPath = os.path.join( os.path.split(__file__)[0], 'ilastik-style.qss' )
-    with file( styleSheetPath, 'r' ) as f:
+    with open( styleSheetPath, 'r' ) as f:
```

Comment:
Actually file has been removed in python 3 so runtime execution will fail.  
yes, I missed to fix other instances!